netlify.toml

```
@@ -2,7 +2,11 @@
     publish = "public"
 
 [build.environment]
-    HUGO_VERSION = "0.145.0"
+    # Consult the Docs WG before bumping Hugo.
+    # Specifically, the version here should match docker/build-tools/Dockerfile in istio/tools,
+    # and the new build container should have been updated by Automator in istio/test-infra
+    # before manually updating the version used by Netlify.
+    HUGO_VERSION = "0.147.2"
```

```suggestion
    HUGO_VERSION = "0.147.4"
```

Bumping to match istio/tools PR